manim/mobject/geometry.py

```diff
@@ -402,9 +387,25 @@ def __init__(self, point=ORIGIN, **kwargs):
 
 
 class SmallDot(Dot):
-    CONFIG = {
-        "radius": DEFAULT_SMALL_DOT_RADIUS,
-    }
+    CONFIG = {"radius": DEFAULT_SMALL_DOT_RADIUS}
+
+
+class AnnotationDot(VMobject):
+    def __init__(self, color=BLUE):
+        VMobject.__init__(self)
+        sourunding_dot = Dot().scale(1.3).set_fill(color=WHITE).set_z_index(-1)
+        innerdot = Dot().set_color(color)
+        annotation_dot = VGroup(sourunding_dot, innerdot)
+        self.add(annotation_dot)
+
+
+class LabeledDot(Mobject):
+    def __init__(self, label_string):
+        Mobject.__init__(self)
+        from manim import MathTex
```

Comment:
Question: I think this should go at the top, no?

Reply:
Answer: for me, it felt more convenient to import this here, because all the other classes in geometry do not need MathTex

Reply:
I don't think that's a good reason to put an import statement inside an init method.

A good reason is to think about the order of initialization of the library. In manim/__init__.py, the `geometry` module is imported before the `tex` modules. So the only way of using MathTex within a class defined in `geometry` is by importing it during runtime (e.g. in the init method of some class, as is being done here), and not during initialization (e.g. at the top of the file). 

Alternatively, we could modify manim/__init__.py to import the `tex` modules before importing the `geometry` module. In that case, it literally does not matter where the import happens because the imported module will already be initialized. Though in that case I would say it'd be better to put it at the top, since there's little to no difference, and it is more standard.

In this case though, I'm torn. I'd go with the latter TBH. @eulertour ?

Reply:
ok, then I will put it on the top

Reply:
When I put 
`from .. import MathTex`
on the top in geometry.py, I get an error:
`ImportError: cannot import name 'MathTex' from partially initialized module 'manim' (most likely due to a circular import) (/home/projects/manim-community/manim/__init__.py)`
so I would for now leave it here.
Or do we want to modify modify manim/init.py ?